integration_tests/__tests__/__snapshots__/multi_project_runner.test.js.snap

```diff
@@ -10,29 +10,31 @@ Ran all test suites.
 `;
 
 exports[`can pass projects or global config 2`] = `
-"Test Suites: 2 passed, 2 total
-Tests:       2 passed, 2 total
+"Test Suites: 3 passed, 3 total
+Tests:       3 passed, 3 total
 Snapshots:   0 total
 Time:        <<REPLACED>>
-Ran all test suites in 2 projects.
+Ran all test suites in 3 projects.
 "
 `;
 
 exports[`can pass projects or global config 3`] = `
-"PASS  project1/__tests__/file1.test.js
+"PASS   BACKEND  project1/__tests__/file1.test.js
+PASS   UI  project3/__tests__/file1.test.js
 PASS  project2/__tests__/file1.test.js"
 `;
 
 exports[`can pass projects or global config 4`] = `
-"Test Suites: 2 passed, 2 total
-Tests:       2 passed, 2 total
+"Test Suites: 3 passed, 3 total
+Tests:       3 passed, 3 total
 Snapshots:   0 total
 Time:        <<REPLACED>>
-Ran all test suites in 2 projects.
+Ran all test suites in 3 projects.
 "
 `;
 
 exports[`can pass projects or global config 5`] = `
-"PASS  project1/__tests__/file1.test.js
+"PASS   BACKEND  project1/__tests__/file1.test.js
+PASS   UI  project3/__tests__/file1.test.js
```

Comment:
Ohh, now I get this change that @thymikee was making for snapshots. Can we merge that one for the next alpha release? We can break the snapshot format for 21, and update the snapshot version. What do you think?